Description: fix FTBFS with gcc 4.7 by including a missing header. This fixes a
 'STDIN_FILENO' and 'STDOUT_FILENO' not declared error.
Author: Fabrice Coutadeur <coutadeurf@gmail.com>
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -25,6 +25,7 @@
 #include <clocale>
 #include <string>
 #include <list>
+#include <unistd.h>
 
 #ifdef HAVE_LIB_AO
 #include <ao/ao.h>
